python-sdk/src/astro/databases/snowflake.py

```diff
@@ -717,11 +728,11 @@ def _build_merge_sql(
         )
 
         merge_target_dict = {
-            f"merge_clause_target_{i}": f"{target_table_name}.{x}"
+            f"merge_clause_target_{i}": f'{target_table_name}."{x}"'
```

Comment:
Why this change?

Reply:
We need to keep identifiers in double quotes.